Documentation/doc/Documentation/Installation.txt

```diff
@@ -334,6 +336,24 @@ cmake -DCGAL_HEADER_ONLY=ON . # configure \cgal
 
 and we do not need to run `make` anymore.
 
+\subsubsection subsection_headeronly_without_configuration Header-only without CMake Configuration
+
+Since \cgal 4.12, \cgal can be used in header-only mode, without even
+configuring \cgal\. The program using \cgal (example, test, demo, etc.)
+must be directly configured using CMake in any case, but then \cgal will be
```

Comment:
I would replace _in any case, but then_ by _, and_

Reply:
I suggest "using CMake. In this case, \cgal..." (because "but" sounds like a drawback)

(sorry for duplicate comments but my last one disappeared...?)